colossalai/kernel/triton/fused_layernorm.py

```diff
@@ -41,7 +40,7 @@ def _layer_norm_fwd_fused(
         for off in range(0, N, BLOCK_SIZE):
             cols = off + tl.arange(0, BLOCK_SIZE)
             x = tl.load(X + cols, mask=cols < N, other=0.0).to(tl.float32)
-            x = tl.where(cols < N, x - mean, 0.0)
```

Comment:
Then `mean` seems not to be used in this kernel. Remove the logics of computing mean (unused calculations)